integration-tests/utils/awsClients.ts

```diff
@@ -0,0 +1,16 @@
+import { CloudWatchLogsClient } from '@aws-sdk/client-cloudwatch-logs'
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'
+import { SQSClient } from '@aws-sdk/client-sqs'
+
+const region = 'eu-west-2'
```

Comment:
We don't need to set this variable as there is a constant in the `awsParameters.ts` file.

Reply:
I've removed that file completely and separated the clients.